pkgs/development/python-modules/sqlcipher3/default.nix

```
@@ -0,0 +1,22 @@
+{ buildPythonPackage, fetchPypi, lib, sqlcipher }:
+
+buildPythonPackage rec {
+  pname = "sqlcipher3";
+  version = "0.5.0";
+
+  propagatedBuildInputs = [ sqlcipher ];
+
+  doCheck = true;
```

```suggestion
  checkInputs = [
    pytestCheckHook
  ];
```

`doCheck` is always true in `buildPythonPackage`.